bundles/framework/featuredata/view/FeatureDataContainer.jsx

```diff
@@ -100,7 +100,7 @@ const createFeaturedataGrid = (features, selectedFeatureIds, showSelectedFirst,
 const createColumnSettings = (selectedFeatureIds, showSelectedFirst, sorting, visibleColumnsSettings) => {
     const { allColumns, visibleColumns, activeLayerPropertyLabels } = visibleColumnsSettings;
     return allColumns
-        .filter(key => !FEATUREDATA_DEFAULT_HIDDEN_FIELDS.includes(key) && visibleColumns.includes(key))
+        .filter(key => (!FEATUREDATA_DEFAULT_HIDDEN_FIELDS.includes(key) || key === ID_FIELD) && visibleColumns.includes(key))
```

Comment:
This is repeated so many times we should probably just add an export for something like `isFieldVisible(field, visibleColumns)` FeatureDataPluginHandler. Then we could comment on it why is ID_FIELD both in FEATUREDATA_DEFAULT_HIDDEN_FIELDS and handled separately?

Reply:
Turns out we don't actually need the (historical) hidden_fields implementation for anything anymore. Removed that and it made things a whole lot simpler.

Also, didn't see the point of artificially cleaning out the id field from the list of properties in "select by properties"-dialog so selecting by id-field is now allowed.